src/Routes.tsx

```diff
@@ -1,84 +1,33 @@
 import React, { FunctionComponent } from 'react';
-import { Redirect, Route, Switch } from "react-router-dom";
-import About from "./components/About";
-import Docs from "./components/Docs";
+import { useLocation } from "react-router-dom";
+import { WorkspaceInfo } from './AppContainer';
+import About from './components/About';
+import Docs from './components/Docs';
 import HitherJobMonitor from './components/HitherJobMonitor/HitherJobMonitor';
 import Config from './containers/Config';
 import Home from "./containers/Home";
-import ImportRecordings from "./containers/ImportRecordings";
-import ImportSortings from "./containers/ImportSortings";
-import RecordingView from "./containers/RecordingView";
-import SortingUnitView from './containers/SortingUnitView';
-import SortingView from './containers/SortingView';
-import TimeseriesForRecordingView from "./containers/TimeseriesForRecordingView";
 
-const TestPage = () => {
-    return <div>{`Test page`}</div>;
-}
+const Routes: FunctionComponent<{width: number, height: number, workspaceInfo: WorkspaceInfo}> = ({width, height, workspaceInfo}) => {
+    const location = useLocation()
```

Comment:
In this file, I have simplified the routing. The old `<Switch><Route ... /></Switch>` method is sort of elegant, but I needed something more explicit.
I continue to discover that hooks are the way to go.
Here `useLocation()` is a really nice way to get the current page URL (aka route, aka location) (whether it is a virtual route or a non-virtual route).